pywbem/_cim_types.py

```diff
@@ -817,26 +817,37 @@ def __hash__(self):
 class CIMInt(CIMType, int):
     """
     Base type for CIM integer data types. Derived from :class:`~pywbem.CIMType`
-    and :class:`py3:int` (for Python 3) or :class:`py2:long` (for Python 2).
+    and :class:`py3:int`. The input parameters consistent with :class:`py3:int`
+    except that `x` may be either a positional or keyword input parameter.
 
     This class has a concept of a valid range for the represented integer,
     based upon the capability of the CIM data type as defined in
     :term:`DSP0004`. The additional constraints defined by possible MinValue
     or MaxValue qualifiers are not taken into account at this level.
 
     The valid value range is enforced when an instance of a subclass of this
-    class (e.g. :class:`~pywbem.Uint8`) is created. Values outside of the
-    valid range raise a :exc:`ValueError`.
-    The enforcement of the valid value range can be disabled via the
-    configuration variable :data:`~pywbem.config.ENFORCE_INTEGER_RANGE`.
+    class (e.g. :class:`~pywbem.Uint8`) is created. Values outside of the valid
+    range raise a :exc:`ValueError`. Enforcement of the valid value range can
+    be disabled via the configuration variable
+    :data:`~pywbem.config.ENFORCE_INTEGER_RANGE`.
 
     Two objects of subclasses of this base class compare equal if their numeric
     values compare equal. Objects of this class are immutable and
     :term:`hashable`, with the hash value being based on its numeric value.
 
+    Parameters:
+          x (:term:`integer` or :term:`string`):
+            A number or a string that can be converted into an integer number.
+            The parameter 'x' can be either a positional or keyword argument
```

Comment:
Any parameter can be passed as a keyword arg or positional arg (except if the `*` is used to require passing as keyword arg, or the `/` is used to require passing as positional, both of which we do not use so far).

If we mention that explicitly, I think we need to give a hint as to why we do that, eg. by adding "for compatibility with Python<=3.7."

Reply:
Or more like ‘to maintain the interface constantly despite the changes to the Python int() interface’.

Reply:
That's better.